csharp/ql/src/semmlecode.csharp.dbscheme

```diff
@@ -480,8 +480,8 @@ case @nullability.kind of
 | 2 = @annotated
 ;
 
-#keyset[nullability, index]
-nullability_member(int nullability: @nullability ref, int index: int ref, int child: @nullability ref)
+#keyset[parent, index]
+nullability_parent(int nullability: @nullability ref, int index: int ref, int parent: @nullability ref)
```

Comment:
Add `unique` in front of `int nullability: @nullability ref`.

Reply:
They're not unique! The same nullability can have many parents. e.g. the nullability of `!` has parents `<!,!>!` and `<!,!,!>!`. So I could have pushed back on this suggestion, as calling something `nullability_parent` suggests a single parent, when really you only think about navigating from a nullabilty to one of its children. Still I've renamed it to `nullability_parent` so that might as well stay.

Reply:
You are right! Sorry about the confusion.